src/arrayMethodSort.js

```diff
@@ -4,8 +4,21 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
+    compareFunction = (a, b) => String(a) > String(b),
+  ) {
+    for (let i = 0; i < this.length; i++) {
+      for (let j = i + 1; j < this.length; j++) {
+        if (compareFunction(this[i], this[j]) > 0) {
```

Comment:
The comparison function should return a number less than 0, 0, or greater than 0. It is not guaranteed that the function will only return 1 or -1. To ensure correct behavior, use `compareFunction(this[i], this[j]) > 0` instead of `compareFunction(this[i], this[j])`.